engine/src/main/java/org/terasology/telemetry/logstash/TelemetryLogstashAppender.java

```diff
@@ -32,12 +31,13 @@
 import net.logstash.logback.stacktrace.ShortenedThrowableConverter;
 import org.slf4j.LoggerFactory;
 import org.terasology.context.Context;
-
+import org.terasology.module.sandbox.API;
 
 /**
  * This is a logback Logstash appender that enriches error logs and sent them to the Logstash in server.
  * The constructor has the default configuration of this appender. The destination of this appender will be set when error reporting is enabled.
  */
+@API
```

Comment:
Is this one safe? I'm not sure if this class extending `LogstashTcpSocketAppender` would allow a malicious module to start messing around with something network related. As an alternative you could perhaps provide a wrapper method somewhere that indirectly lets a user request a new appender or something.